pkg/collector/container_hc_collector.go

```diff
@@ -125,31 +130,21 @@ func (c *Collector) updateBPFMetrics() {
 		isSystemProcess := containerID == c.systemProcessName
 
 		c.createContainersMetricsIfNotExist(containerID, ct.CGroupID, ct.PID, config.EnabledEBPFCgroupID)
+		c.ContainersMetrics[containerID].PID = ct.PID
 		comm := C.GoString((*C.char)(unsafe.Pointer(&ct.Command)))
 		// System process is the aggregation of all background process running outside kubernetes
 		// this means that the list of process might be very large, so we will not add this information to the cache
 		if !isSystemProcess {
 			c.ContainersMetrics[containerID].SetLatestProcess(ct.CGroupID, ct.PID, comm)
 		} else if config.EnableProcessMetrics {
 			c.createProcessMetricsIfNotExist(ct.PID, comm)
-			if err := c.ProcessMetrics[ct.PID].CPUTime.AddNewDelta(ct.ProcessRunTime); err != nil {
+			err := c.ProcessMetrics[ct.PID].CPUTime.AddNewDelta(ct.ProcessRunTime)
+			if err != nil {
 				klog.V(5).Infoln(err)
 			}
 		}
 
 		c.ContainersMetrics[containerID].CurrProcesses++
-		// system process should not include container event
```

Comment:
are we able to remove this part?

Reply:
Yes, because we read those metrics in the `updateCgroupMetrics()`

Reply:
@sunya-ch is it ok only keep the `BytesRead` and `BytesWrite` metrics only in the `CgroupFSStats` metrics?

Reply:
@sunya-ch , could you please confirm this? once @sunya-ch looks good, we can go ahead with merge.